### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by azoitl**
Reworked CONTRIBUTING.md to the processes of Github.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,73 +1,160 @@
...
-Eclipse 4diac™ in its current form has been started 2007 as an open source
-project fostering the further development of IEC 61499 for its use in
-distributed Industrial Process Measurement and Control Systems (IPMCS) and to
-further distribute research results from the original contributors. From the
-beginning, it has provided everything that is necessary to program and execute
-distributed IPMCS.
+[Eclipse 4diac](https://eclipse.dev/4diac) is a Open Source Framework for Industrial Automation & Control. 
+It is a reference implementation for the IEC 61499 standard. 
+IEC 61499 defines a domain specific modeling language for developing distributed industrial control solutions.
+IEC 61499 extends IEC 61131-1 by improving the encapsulation of software components for increased re-usability, providing a vendor independent format, and simplifying support for controller to controller communication.
```
Comment (MartinMelikMerkumians) [inline CONTRIBUTING.md]: 61131-1 describes the hardware of a plc as far as I remember. Do you mean part 3 here or do you want to refer to all parts?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,73 +1,160 @@
-# Contributing to Eclipse 4DIAC
+# Contributing to 4diac FORTE
```
Comment (moniwe) [inline CONTRIBUTING.md:1]: maybe Eclipse 4diac FORTE?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,73 +1,160 @@
...
+Fixes https://github.com/eclipse-4diac/4diac-forte/issues/321 
+```
 
 ## Contact
 
 Contact the project developers via the project's "dev" list.
 
-* https://dev.eclipse.org/mailman/listinfo/4diac-dev
\ No newline at end of file
+* https://dev.eclipse.org/mailman/listinfo/4diac-dev
```
Comment (moniwe) [inline CONTRIBUTING.md:160]: maybe add the license section at the end?

**Review by moniwe — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (azoitl) [reply CONTRIBUTING.md:1]: We normally use Eclipse 4diac for the whole open source project and 4diac FORTE and 4diac IDE for the two main products provided by Eclipse 4diac. But we never really write Eclipse in front of the products.

File: CONTRIBUTING.md (hunk 3 above)
Comment (azoitl) [reply CONTRIBUTING.md:160]: maybe. feel free to do in another PR.

File: CONTRIBUTING.md (hunk 1 above)
Comment (azoitl) [reply CONTRIBUTING.md]: yes this is wrong changed to 61131-3

**Review by MartinMelikMerkumians — APPROVED** [review]

**Pull request merged**